src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -23,20 +23,41 @@ const PromoListComponent = ({ promoItems, dir }) => {
   const viewRef = useViewTracker(eventTrackingData.block);
   const { isAmp } = useContext(RequestContext);
 
-  const StoryPromo = isAmp || isLive() ? _StoryPromo : FrostedGlassPromo;
+  const promoVariation = useOptimizelyVariation(
+    'high_impact_feature_analysis_promo',
+  );
+
+  let hasFrostedGlassPromo = false;
+
+  if (promoVariation) {
+    if (promoVariation === 'variation_1' && !isAmp) {
+      hasFrostedGlassPromo = true;
+    }
+  }
```

Comment:
Isn't this equivalent?
```suggestion
  const hasFrostedGlassPromo = promoVariation === 'variation_1' && !isAmp;
```